io/disk/arcconf/arcconf_raid_oper.py.data/Readme

```diff
@@ -0,0 +1,14 @@
+arcconf - Command line interface to configure PMC-Sierra
+(Microsemi) Controllers.
+
+arcconf_raid_oper.py covers functionality like RAID creation,
+deletion, Physical disk & logical disk format, and gather
+controller status using getconfig.
```

Comment:
@PraveenPenguin my view is, it is ok to have redundant information in docstring and readme file. People look into readme file more than docstring, IMO.